src/dotnet/APIView/APIViewWeb/Pages/Shared/_CommentThreadPartial.cshtml

```diff
@@ -21,20 +21,16 @@
                             <div class="comment-actions">
                                 <form data-post-update="comments" asp-controller="Comments" asp-action="ToggleUpvote" method="post" asp-route-reviewId="@Model.ReviewId">
                                     <input type="hidden" name="commentId" value="@comment.CommentId" />
-                                    @{
-                                        var upvoteclass = "btn-upvote";
-                                        //btn-upvote shows btn in grey when not active
-                                        //not adding this class to show upvotes clearly when someoen else upvoted
-                                        if (comment.Upvotes.Any() && !comment.Upvotes.Contains(User.GetGitHubLogin())) {
-                                            upvoteclass = "";
+                                    <button type="submit" class="btn btn-light btn-sm btn-upvote" title="@string.Join(", ", comment.Upvotes)">
```

Comment:
You still need the active-if="@comment.Upvotes.Contains(User.GetGitHubLogin())" here

Reply:
Sure. thanks for catching it.